boards/renesas/ek_ra6m1/doc/index.rst

```diff
@@ -94,6 +94,8 @@ The below features are currently supported on Zephyr OS for EK-RA6M1 board:
 +-----------+------------+----------------------+
 | CLOCK     | on-chip    | clock control        |
 +-----------+------------+----------------------+
+| COUNTER   | on-chip    | counter              |
```

Comment:
Wrong commit message, should start with `boards: renesas:...`

Reply:
Thank you, I have changed the commit message and split this modification into a commit